master/packaging/master.yaml

```diff
@@ -19,6 +19,9 @@
 ## dynamic agent types.
 #provisioner:
 
+## A user friendly name to identify this cluster by.
+#  cluster_name: Dev
```

Comment:
I think this field in the wrong place (under provisioner) and has the wrong indent.

Reply:
you're right. It also seems like our indentation wrt to commented examples are a bit in consistent, although not necessarily wrong. 

sometimes `#` sings are padded with a space and sometimes they're not.but as long as they're padded with only a single space it should be fine.